tasks/proposeUpgrade.ts

```diff
@@ -0,0 +1,27 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types/runtime'
+
+require('@openzeppelin/hardhat-upgrades')
+require('dotenv').config()
+
+const addresses: { [key: string]: string } = {
```

Comment:
Are these basically the current addresses? If so, how would this change after upgrades?

Reply:
They are the addresses of the proxies, which won't change